vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -101,6 +106,12 @@ def __init__(self, quant_config: Fp8Config):
         self.quant_config = quant_config
         self.cutlass_fp8_supported = cutlass_fp8_supported()
 
+        # For GPUs that lack FP8 hardware support, we can leverage the Marlin
+        # kernel for fast weight-only FP8 quantization
+        capability = torch.cuda.get_device_capability()
```

Comment:
Please use the stateless version that Kaichao added

Reply:
Thanks, I didn't notice `current_platform.get_device_capability()`